Code/Tools/ProjectManager/Resources/ProjectManagerCompletionTags.txt

```diff
@@ -0,0 +1,46 @@
+AWS
+Achievements
+Animation
+Assets
+AtomShader
+Audio
+Barrier
+Compression
+Content
+Core
+Creation
+DCC
+Debug
+Design
+Digital
+Editor
+Environment
+Framework
+Gameplay
+Input
+InverseKinematics
+Lighting
+Materials
+Multiplayer
+Navigation
+Network
+PBR
+Physics
+Profiler
+RemoteTools
+Rendering
+SDK
+Sample
+ScriptAutomation
+Scripting
+Simulation
+Slices
+Stars
```

Comment:
Should check with UX on this list whether we want a large suggestion list or not, a couple of entries are quite specific that are not going to be used that often.